Source/Urho3D/Graphics/Direct3D11/D3D11Graphics.cpp

```diff
@@ -2002,9 +2002,11 @@ bool Graphics::CreateDevice(int width, int height)
     if (!impl_->device_)
     {
         UINT deviceFlags = D3D11_CREATE_DEVICE_BGRA_SUPPORT;
-#ifdef _DEBUG
-        deviceFlags |= D3D11_CREATE_DEVICE_DEBUG;
-#endif
+        if (screenParams_.gpuDebug_)
+        {
+            // Enable the debug layer if requested.
+            deviceFlags |= D3D11_CREATE_DEVICE_DEBUG;
```

Comment:
Does OpenGL have a comparable flag?

Reply:
I'm not aware of such flag for OpenGL. Vulkan does have debug layer that we can enable in runtime.

The point of the change is to make this switch available in runtime, not in compile time.